docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
+6. **bot_message**: previous message from a bot/agent. This is an important parameter, many times the entity value relies on the message from the bot/agent i.e. what is bot saying? or asking?
 
-    - *CURL Output:*
+   For example, bot might ask for departure date to book a flight and user might reply with a date. Now, it is difficult to disambiguate whether its departure date or arrival date unless, we know what bot is asking for?
 
-      ```json
-      {
-        "data": [
-          {
-            "detection": "message",
-            "original_text": "9049961794",
-            "entity_value": {
-              "value": "9049961794"
-            }
-          }
-        ]
-      }
-      ```
+   ```
+   bot: Please help me with date of departure?
+   user: 23rd March
+   ```
 
-  - Example 2: 
+### **Output Format**
 
-    - ```python
-      message = 'Please call me'
-      entity_name = 'phone_number'
-      structured_value = None
-      fallback_value = '9049961794'
-      bot_message = None
-      ```
+The output of entities detected will be stored in a list of dictionary containing the following structure:
 
-    - *Python:* 
+```
+[
+    {
+        "entity_value": entity_value,
+        "detection": detection_method,
+        "original_text": original_text
+    },
+]
+```
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_phone_number
-      output = get_phone_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+Consider the following example for detailed explanation:
 
-    - *CURL command:*
+```
+"I want to order from mcd"
+```
 
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
+- entity_value: This will store the value of entity (i.e entity value) that is detected. The output will be in dictionary format. For example, `{"value": "McDonalds"}`.
+- detection: This will store how the entity is detected i.e. whether from *message*, *structured value* or *fallback value*.
+- original_text: This will store the actual value that is detected. For example, mcd.
 
-      ​
+```python
+Example 1:
+input:  message = 'i want to order chinese from  mainland china and pizza from domminos'
+        entity_name = 'restaurant'
+        structured_value = None
+        structured_value_verification = 0
+        fallback_value = None
+        bot_message = None
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/phone_number/?message=Please%20call%20me&entity_name=phone_number&structured_value=&fallback_value=9049961794&bot_message='
-      ```
+output:
+[
+  {
+    "detection": "message",
+    "original_text": "mainland china",
+    "entity_value": { "value": "Mainland China" }
+  },
+  {
+    "detection": "message",
+    "original_text": "domminos",
+    "entity_value": { "value": "Domino's Pizza"}
+  }
+]
 
-    - *CURL Output:*
+Example 2:
+input:  message = 'i wanted to watch movie'
```

Comment:
Please use some other example. Preferably date. All across it seems as if we are using only dish, cuisine and restaurants.